plugins.json

```diff
@@ -1,4 +1,111 @@
 [
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-github",
+    "icon_data": "data:image/svg+xml;base64,PHN2ZyByb2xlPSJpbWciIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48dGl0bGU+R2l0SHViIGljb248L3RpdGxlPjxwYXRoIGQ9Ik0xMiAuMjk3Yy02LjYzIDAtMTIgNS4zNzMtMTIgMTIgMCA1LjMwMyAzLjQzOCA5LjggOC4yMDUgMTEuMzg1LjYuMTEzLjgyLS4yNTguODItLjU3NyAwLS4yODUtLjAxLTEuMDQtLjAxNS0yLjA0LTMuMzM4LjcyNC00LjA0Mi0xLjYxLTQuMDQyLTEuNjFDNC40MjIgMTguMDcgMy42MzMgMTcuNyAzLjYzMyAxNy43Yy0xLjA4Ny0uNzQ0LjA4NC0uNzI5LjA4NC0uNzI5IDEuMjA1LjA4NCAxLjgzOCAxLjIzNiAxLjgzOCAxLjIzNiAxLjA3IDEuODM1IDIuODA5IDEuMzA1IDMuNDk1Ljk5OC4xMDgtLjc3Ni40MTctMS4zMDUuNzYtMS42MDUtMi42NjUtLjMtNS40NjYtMS4zMzItNS40NjYtNS45MyAwLTEuMzEuNDY1LTIuMzggMS4yMzUtMy4yMi0uMTM1LS4zMDMtLjU0LTEuNTIzLjEwNS0zLjE3NiAwIDAgMS4wMDUtLjMyMiAzLjMgMS4yMy45Ni0uMjY3IDEuOTgtLjM5OSAzLS40MDUgMS4wMi4wMDYgMi4wNC4xMzggMyAuNDA1IDIuMjgtMS41NTIgMy4yODUtMS4yMyAzLjI4NS0xLjIzLjY0NSAxLjY1My4yNCAyLjg3My4xMiAzLjE3Ni43NjUuODQgMS4yMyAxLjkxIDEuMjMgMy4yMiAwIDQuNjEtMi44MDUgNS42MjUtNS40NzUgNS45Mi40Mi4zNi44MSAxLjA5Ni44MSAyLjIyIDAgMS42MDYtLjAxNSAyLjg5Ni0uMDE1IDMuMjg2IDAgLjMxNS4yMS42OS44MjUuNTdDMjAuNTY1IDIyLjA5MiAyNCAxNy41OTIgMjQgMTIuMjk3YzAtNi42MjctNS4zNzMtMTItMTItMTIiLz48L3N2Zz4=",
+    "download_url": "https://github.com/mattermost/mattermost-plugin-github/releases/download/v1.0.0/github-1.0.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-github/releases/tag/v1.0.0",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl7QCTIACgkQ0bVLR6XO/sS5lBAAkRyZ/fR6i2gNgynY+5YSf0h+SnzTqsmS/9YnQARVsu6PvoHLx+JdpeI8zNHtk3WsKuo2lERy7wk0KmQgGGZdFVagraVZU8gB/8LGAqyBUOOkr1mhBssjdJWoaYz7BNh6UjLrawBQcUqHKuw9L9/TrGuxvZ20If/zP5C4wX9Oe7dKeXQ+6qhelb2WM9z6U7TJQmGAd2xewZHBi4naESnZI5HxOtnwqO9uqZMooXLkk7Xn0EmIZ5dq572YoV9Noe7UF2GBSwaekskI/IjT1qGs5OjGtPX9tHMMQAEtJ83muNd66TAmqSmGT6olTe0+prRyo2V8KBOc1kw6WnitFBkjMTVEEmHFBtGXQGKMWludid9a6ljIT/jfdH+ZWGgFL1JSRoV7++/iDwQO3z7HW7jfrnwiSqdP575tvt9jdgkNQHWkBdXupGFjErWpuFaPR0frpGe/BzqwbjdiWUkBWyR/MDY1XbA43X4gyh8RSa4MHzPCUYfXoX3wJ6Y1HvT6x5si57e3GIooAH9Ebj3OE1Y/bEDJ419J5/yLwe7q0KGSQhYMd2spAgIgH/+PAbJ5W9uam4Gkb3PJ5CGr6n68WBWujLWoh5SCV0jipoiU2GvyfV/kJytl9JKfC82snslSoOfK76fh71pg2PGIy8X0460b46ElxfOjRj9vnvIjP+QeptE=",
+    "manifest": {
+      "id": "github",
+      "name": "GitHub",
+      "description": "GitHub plugin for Mattermost.",
```

Comment:
```suggestion
      "description": "GitHub plugin for Mattermost",
```

Reply:
@aaronrothschild Actually, most of the plugin descriptions end with a dot. Should this be holistically changed?
![Screenshot from 2020-05-28 21-09-13](https://user-images.githubusercontent.com/16541325/83183057-8b3c4200-a127-11ea-8fed-01fbf4614e01.png)